board/at91sam9m10g45ek/at91sam9m10g45ek.c

```diff
@@ -132,34 +132,86 @@ static void recovery_buttons_hw_init(void)
 }
 #endif /* #if defined(CONFIG_NANDFLASH_RECOVERY) || defined(CONFIG_DATAFLASH_RECOVERY) */
 
-static int ek_special_hw_init(void)
+#ifdef CONFIG_USER_HW_INIT
+//*----------------------------------------------------------------------------
+//* \fn    nedap_usb_fix
+//* \brief This function fixes an issue with the usb
+//*----------------------------------------------------------------------------*/
+static void nedap_usb_fix(void)
 {
-	/*
-	 * For on the sam9m10g45ek board, the chip wm9711 stay in the test mode,
-	 * so it need do some action to exit mode.
-	 */
-	const struct pio_desc wm9711_pins[] = {
-		{"AC97TX", AT91C_PIN_PD(7), 0, PIO_PULLUP, PIO_OUTPUT},
-		{"AC97FS", AT91C_PIN_PD(8), 0, PIO_PULLUP, PIO_OUTPUT},
-		{(char *)0, 0, 0, PIO_DEFAULT, PIO_PERIPH_A},
+	unsigned int lock = 1000000;
```

Comment:
`grep -r "unsigned int" | wc -l` --> 1169
`grep -r uint | wc -l` --> 52
So they mixed it a bit.
I suggest to stay with `unsigned int`...